apps/condo/domains/organization/schema/OrganizationEmployee.js

```diff
@@ -141,6 +141,14 @@ const OrganizationEmployee = new GQLListSchema('OrganizationEmployee', {
             if (!hasDbFields(['organization'], resolvedData, existingItem, context, addValidationError)) return
             if (!hasOneOfFields(['email', 'name', 'phone'], resolvedData, existingItem, addValidationError)) return
         },
+        afterChange: async ({ context, operation, existingItem, updatedItem }) => {
+            const isSoftDeleteOperation = operation === 'update' && !existingItem.deletedAt && Boolean(updatedItem.deletedAt)
+
+            // TODO(DOMA-4440): we need to make a tool for automatic cascading soft deletion of related objects
```

Comment:
It's also will be good if we can fix our `MODELS.PROTECT` and other staff, and then auto-soft delete just like hard-delete